javascript/array-methods-2/js-array-methods-2_animal-strings/index.js

```diff
@@ -39,18 +40,18 @@ const indexOfGiraffe = null;
 // Hint: There is no need to upper-/lowercase the strings before sorting them.
 // Hint: sort() mutates the original array, which is bad.
 // -> Use animals.slice().sort(...) to make a copy.
-const animalsSortedAlphabetically = null;
+const animalsSortedAlphabetically = animalStrings.slice().sort();
 
 // Hint: Guess what? There is a string method called `endsWith()`.
-const anyAnimalEndsWithLetterZ = null;
+const anyAnimalEndsWithLetterZ = animalStrings.some(animal => animal.endsWith('z'))
```

Comment:
I think there is a small mistake with the function variable animal. It should be entered between parenthesis after initiating the function, like `animalStrings.some((animal) => animal.endsWith('z'))`. I'm curious to know if this worked for you as currently written.